src/processing/CommunicateChange.java

```diff
@@ -0,0 +1,90 @@
+package processing;
+
+import java.util.*;
+import processing.IChanges;
+import processing.utility.*;
```

Comment:
No need of `processing.IChanges`. Please confirm.

Reply:
Yes, no need.